<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-P57AQ83" name="Test.Day1" tests="4" failures="0" timestamp="2022-03-05T10:20:24 IST" time="0.015" errors="1">
  <testcase name="Test" time="0.007" classname="Test.Day1"/>
  <system-out/>
  <testcase name="Test" time="0.004" classname="Test.Day1"/>
  <system-out/>
  <testcase name="FirstTest" time="0.000" classname="Test.Day1">
    <error type="org.testng.TestNGException" message="
Parameter &#039;URL&#039; is required by @Test on method FirstTest but has not been marked @Optional or defined
in C:\Users\Vivek\eclipse-workspace\TestNG Tutorial\testng4.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'URL' is required by @Test on method FirstTest but has not been marked @Optional or defined
in C:\Users\Vivek\eclipse-workspace\TestNG Tutorial\testng4.xml
at org.testng.internal.Parameters.createParams(Parameters.java:270)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:358)
at org.testng.internal.Parameters.createParameters(Parameters.java:700)
at org.testng.internal.Parameters.handleParameters(Parameters.java:884)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- FirstTest -->
  <system-out/>
  <testcase name="Test" time="0.004" classname="Test.Day1"/>
  <system-out/>
</testsuite> <!-- Test.Day1 -->
